bash/bash.sh

```diff
@@ -146,7 +152,7 @@ debug() {
 }
 
 currentdir() {
-    pwd | egrep -o '[^/]*' | tail -1
+    basename "$(pwd)"
```

Comment:
The piping and what-have-you wasn't needed, only `basename`. If you don't have it, for some reason, you can always use: `${PWD##*/}`